### PR TITLE
Greyscale houses that change to normal shading on hit

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Take a look and say what you think, it's definitely very distinct, I think with gray sidewalks it may look better.

I'm not completely sold either way, but I think the benefit of it being obvious which houses to hit is worth it.

### CODE REVIEW

**Review by joncatanio — APPROVED** [review]
This looks good and functionally works. I am a little torn with it though. Is there anyway we can like start with the colorful view and while the camera moves to the car we also interpolate the colors to their greyscale values? So it's like a visual queue of what the player is working toward?

**Comment by rgarmsen2295** [comment]
Yeah I get what you're saying, in a perfect world I'd love to have the camera pan around the world and then have the color slowly drain out and then throw the camera behind the car.  There's probably a way to get it to do what you're describing in a super basic sense but a) would take a fair bit of work to get that interpolation I think and even if that was easy b) the camera moves super quickly from start to the car so i don't know how much the player would even see.

Great idea, just the timing is what I'm more worried about...we could reduce the grayscale effect so it's less pronounced maybe, but that also goes against the purpose :/

**Comment by rgarmsen2295** [comment]
Okay, this has been updated pretty heavily.

Once the player hits start on the main menu the camera pans over to the start position as normal.  Then the houses start to interpolate to greyscale coloring from their usual coloring.  Once they are fully grayscale, the player's car spawns and they are allowed to move (and the timer starts counting down).

I also commented out the marker arrows as they seemed kind of redundant, but can easily uncomment them if we still want them as well.

Let me know final thoughts and whether we want this or not, also, whether specific things (e.g not drawing the player at the start) are what we want.

File: src/GameObject.h [hunk 1]
```diff
@@ -39,6 +39,9 @@ class GameObject : public std::enable_shared_from_this<GameObject> {
    // Object is being fractured
    bool fracture;
 
+   // Can the player move (hack hack hack)
```
Comment (joncatanio) [inline src/GameObject.h:42]: hack hack hack all day long!

File: src/Program.cpp [hunk 2]
```diff
@@ -72,6 +72,9 @@ void Program::addDefaultAttributesAndUniforms() {
 	// TODO(rgarmsen2295): Remove from default attributes
 	addUniform("billboardTransform");
 
+	// I'd give this a TODO to remove but I don't think this is gonna change at this point
```
Comment (joncatanio) [inline src/Program.cpp:75]: LOL

**Pull request merged**